src/lib.rs

```diff
@@ -169,6 +169,9 @@ pub struct ClientBuilder<P> {
     max_response_size: u32,
     retry_policy: P,
     ping_config: Option<PingConfig>,
```

Comment:
I think the same applies for client initiated pings, can you add #[cfg(feature = "native")] here as well?